components/core/src/fs.rs

```diff
@@ -201,13 +201,39 @@ pub fn find_command(command: &str) -> Option<PathBuf> {
                 if candidate.is_file() {
                     return Some(candidate);
                 }
+                else {
+                    match find_command_with_pathext(&candidate) {
```

Comment:
Nope, you're right. I looked quick and didn't notice we were in an iterator and trying to short-circuit out of it in this function as well